.github/workflows/python-package.yml

```diff
@@ -5,35 +5,46 @@ name: Python package
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
+      - "release/**"
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+      - "release/**"
+  workflow_dispatch:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Might be worth to keep 3.7 if it's widely used

Reply:
Yeah good point. I don't have figures around that, but if you think it's worth it we can keep it a little bit longer.

Reply:
And it's actually also mentioned in the README.md file, so better keeping it 👍